addons/mail/static/src/models/file_uploader_view/file_uploader_view.js

```diff
@@ -49,7 +49,7 @@ registerModel({
             if (this.activityView) {
                 return replace(this.activityView.activity.thread);
             }
-            if (this.attachmentBoxView) {
+            if (this.attachmentBoxView && this.attachmentBoxView.chatter.thread) {
```

Comment:
why change?

Reply:
Same reason as with https://github.com/odoo/odoo/pull/82698#discussion_r784352069
Had actual crash in tests and turns out `Chatter/thread` can be unset, so we have to guard it otherwise it does `replace(undefined)` which raises an error.